tasks/utils.js

```diff
@@ -108,6 +111,14 @@ function copyDir(srcPath, dstPath) {
     if (isDir(srcPath)) fs.copySync(srcPath, dstPath);
 }
 
+function zipDir(srcPath, dstPath) {
+    if (isDir(srcPath)) {
```

Comment:
it would be better if you call fatalError if not a dir, for then the caller's intended action can't be done